bigbluebutton-html5/imports/ui/components/whiteboard/slide-controls/component.jsx

```diff
@@ -0,0 +1,296 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './styles.scss';
+import Button from '/imports/ui/components/button/component';
+import { callServer } from '/imports/ui/services/api/index.js';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+
+export default class SlideControls extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { sliderValue: 100 };
+
+    this.nextSlide = this.nextSlide.bind(this);
```

Comment:
Tested it, you guys were right in most of cases.
It's just this.skipToSlide doesn't need any bindings, and.. well, an empty clickHandler function.
I actually did it differently here: https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/components/polling/component.jsx#L41
I wonder which way is better.